node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function copy() {
+  try {
+    const source = process.argv[2];
+    const destination = process.argv[3];
+    const info = await readFile(source, 'utf8');
```

Comment:
By using 'utf8' this will only copy text files.

Reply:
Adjusted, thanks for the note!